static/bidder-info/liftoff.yaml

```diff
@@ -1,6 +1,6 @@
 endpoint: "https://rtb.ads.vungle.com/bid/t/c770f32"
 maintainer:
-  email: platform-ssp@liftoff.io
+  email: vxssp@liftoff.io
```

Comment:
@Vungle-GordonTian  Prebid team has sent an mail on `vxssp@liftoff.io` for maintainer email verification

Reply:
Sent. Please check.

Reply:
> Response sent. Please check.

received response